src/repr/src/scalar.rs

```diff
@@ -2611,6 +2669,7 @@ impl<'a> ScalarType {
             ScalarType::Int2Vector => (*INT2VECTOR).iter(),
             ScalarType::MzTimestamp => (*MZTIMESTAMP).iter(),
             ScalarType::Range { .. } => (*RANGE).iter(),
+            ScalarType::MzAclItem { .. } => (*MZACLITEM).iter(),
```

Comment:
Also needs to be added to `enumerate` below.

Reply:
Done